source/data_preprocessing/filterTweets.py

```diff
@@ -0,0 +1,70 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Reads raw twitter output and filters based on a list of regular expressions.
+
+June, 2019
+@author: Joshua Rubin
+"""
+
+import os
+import re
+import json
+
+
+def filterTweets(inputDirectory, outputDirectory, filters):
+    """Reads raw twitter output and filters based on a list of regular expressions.
+    
+    Files are json formatted lists of [tweet, date].
+    inputDirectory:  where to look for files starting with @ containing raw tweets
+    outputDirectory: where to deposit identically named files with filters applied
+    filters:         list of reg. expressions whose matches will be removed
+    """
+
+    # Look for files beginning with @, i.e. twitter handles
+    tweetFiles = [f for f in os.listdir(inputDirectory) if f[0]=='@']
+        
+    for filename in tweetFiles:   
+        
+        outputTweets = []
+
+        print('Processing ' + filename + '.', end=' ')
+        
+        inputPath  = os.path.join(inputDirectory, filename)
+
+        with open(inputPath, 'r') as file:
+            inData =  json.loads(file.read())
+    
+        for tweet in inData:
+               
+            filteredTweet = tweet[0]
```

Comment:
You should make the code in this `for` loop a function. If you have time, I would write some tests for it.